modules/function/src/main/java/com/opengamma/strata/function/StandardComponents.java

```diff
@@ -142,4 +176,33 @@ public static PricingRules pricingRules() {
     return StandardPricingRules.standard();
   }
 
+  /**
+   * Returns the standard calculation functions.
+   * <p>
+   * These rules define how to calculate the standard measures for the standard asset classes.
```

Comment:
"rules" looks a bit out of place given that we're planning to get rid of pricing rules.